components/ca-radio-group.js

```diff
@@ -23,33 +22,38 @@ define([], () => {
          */
         render() {
 
-            if (!this.data || this.data.length === 0) return;
+            if (!this.data) return;
 
             this.destroy();
 
             const container = document.createElement('fieldset');
+            const questionText = document.createElement('legend');
+            const innerContainer = document.createElement('div');
 
-            this.data.forEach(item => {
+            questionText.innerHTML = this.data.description;
```

Comment:
I think we should have a (boolean) attribute to enable use of HTML descriptions with the default being text. That way we can enable it explicitly from the API format string (see ca-form.getSubProperties) and keep ourselves safe from future XSS problems with ca-radio-group when it eventually accidentally gets used for user-generated content.